lib/db.js

```diff
@@ -186,22 +193,40 @@ DB.prototype.get = function(domain, req) {
         return this._getSchema(tableName)
         .then(function(schema) {
             self.schemaCache[tableName] = schema;
-            return self._get(tableName, req, schema);
+            return self._get(tableName, req, schema, true);
         });
     } else {
         return P.try(function() {
-            return self._get(tableName, req, self.schemaCache[tableName]);
+            return self._get(tableName, req, self.schemaCache[tableName], true);
         });
     }
 };
 
+DB.prototype._createGetQuery = function(tableName, req, schema, includePreparedForDelete) {
+    var extracted = dbu.extractGetParams(tableName, req, schema, includePreparedForDelete);
+    var key = stringify(req);
```

Comment:
@gwicke the extractGetParams nulls out all values, but we don't need to put them back, they are not used any more, we already have a array of params we need.